cms/envs/production.py

```diff
@@ -609,3 +609,6 @@ def get_env_setting(setting):
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+############### Settings for edx-rbac  ###############
+SYSTEM_WIDE_ROLE_CLASSES = ENV_TOKENS.get('SYSTEM_WIDE_ROLE_CLASSES') or SYSTEM_WIDE_ROLE_CLASSES
```

Comment:
One small NIT:

Can you move this block to above:
plugin_settings.add_plugins

The last thing in this file should be to load plugin settings, your setting would not be available to plugins